xml/hwbp_registers.xml

```diff
@@ -846,19 +860,17 @@
             and writing it has no effect.
         </field>
         <field name="count" bits="23:10" access="WARL" reset="1">
-            When count is decremented to 0, the trigger fires. Instead of
-            changing \FcsrIcountCount from 1 to 0, it is also acceptable for hardware to
-            clear \FcsrIcountM, \FcsrIcountS, \FcsrIcountU, \FcsrIcountVs, and
-            \FcsrIcountVu. This allows \FcsrIcountCount to be hard-wired
-            to 1 if this register just exists for single step.
+            The trigger will generally fire after \FcsrIcountCount instructions
+            in enabled modes have been executed.
```

Comment:
How about adding "See above for the precise behavior."  Otherwise the "generally" sounds pretty imprecise and the reader might not understand that it's all specified properly.

Reply:
Good idea. Done.